src/Swashbuckle.AspNetCore.Annotations/AnnotationsOperationFilter.cs

```diff
@@ -10,18 +10,38 @@ public class AnnotationsOperationFilter : IOperationFilter
     {
         public void Apply(OpenApiOperation operation, OperationFilterContext context)
         {
-            if (context.MethodInfo == null) return;
+            IEnumerable<object> controllerAttributes = Array.Empty<object>();
+            IEnumerable<object> actionAttributes = Array.Empty<object>();
+            IEnumerable<object> metadataAttributes = Array.Empty<object>();
 
-            var controllerAttributes = context.MethodInfo.DeclaringType.GetCustomAttributes(true);
-            var actionAttributes = context.MethodInfo.GetCustomAttributes(true);
+            if (context.MethodInfo != null)
+            {
+                controllerAttributes = context.MethodInfo.DeclaringType.GetCustomAttributes(true);
+                actionAttributes = context.MethodInfo.GetCustomAttributes(true);
+            }
 
-            // NOTE: When controller and action attributes are applicable, action attributes should take precendence.
-            // Hence why they're at the end of the list (i.e. last one wins)
-            var controllerAndActionAttributes = controllerAttributes.Union(actionAttributes);
+#if NET5_0_OR_GREATER
```

Comment:
```suggestion
#if NET6_0_OR_GREATER
```

Should this only target minimal apps in .NET 6 and above?

Reply:
Yeah, probably. I went with this now now as the 6 SDK wasn't available yet in main and this didn't seem to do any harm.

I can update it once your 6 SDK updates are merged it.